multisig/simulations/vip-002/vip-002-ethereum/simulations.ts

```diff
@@ -0,0 +1,569 @@
+};
+
+interface VTokenState {
+  name: string;
+  symbol: string;
+  decimals: number;
+  underlying: string;
+  exchangeRate: BigNumberish;
+  comptroller: string;
+}
+
+const vTokenState: { [key in VTokenSymbol]: VTokenState } = {
+  // Core Pool
+  vWBTC_Core: {
+    name: "Venus WBTC (Core)",
+    symbol: "vWBTC_Core",
+    decimals: 8,
+    underlying: tokens.WBTC,
+    exchangeRate: parseUnits("1", 18),
+    comptroller: COMPTROLLER_CORE,
+  },
+  vWETH_Core: {
+    name: "Venus WETH (Core)",
+    symbol: "vWETH_Core",
+    decimals: 8,
+    underlying: tokens.WETH,
+    exchangeRate: parseUnits("1", 28),
+    comptroller: COMPTROLLER_CORE,
+  },
+  vUSDT_Core: {
+    name: "Venus USDT (Core)",
+    symbol: "vUSDT_Core",
+    decimals: 8,
+    underlying: tokens.USDT,
+    exchangeRate: parseUnits("1", 16),
+    comptroller: COMPTROLLER_CORE,
+  },
+  vUSDC_Core: {
+    name: "Venus USDC (Core)",
+    symbol: "vUSDC_Core",
+    decimals: 8,
+    underlying: tokens.USDC,
+    exchangeRate: parseUnits("1", 16),
+    comptroller: COMPTROLLER_CORE,
+  },
+  vcrvUSD_Core: {
+    name: "Venus crvUSD (Core)",
+    symbol: "vcrvUSD_Core",
+    decimals: 8,
+    underlying: tokens.crvUSD,
+    exchangeRate: parseUnits("1", 28),
+    comptroller: COMPTROLLER_CORE,
+  },
+  vCRV_Core: {
+    name: "Venus CRV (Core)",
+    symbol: "vCRV_Core",
+    decimals: 8,
+    underlying: tokens.CRV,
+    exchangeRate: parseUnits("1", 28),
+    comptroller: COMPTROLLER_CORE,
+  },
+
+  // Stablecoins Pool
+  vUSDT_Stablecoins: {
+    name: "Venus USDT (Stablecoins)",
+    symbol: "vUSDT_Stablecoins",
+    decimals: 8,
+    underlying: tokens.USDT,
+    exchangeRate: parseUnits("1", 16),
+    comptroller: COMPTROLLER_STABLECOINS,
+  },
+  vUSDC_Stablecoins: {
+    name: "Venus USDC (Stablecoins)",
+    symbol: "vUSDC_Stablecoins",
+    decimals: 8,
+    underlying: tokens.USDC,
+    exchangeRate: parseUnits("1", 16),
+    comptroller: COMPTROLLER_STABLECOINS,
+  },
+  vcrvUSD_Stablecoins: {
+    name: "Venus crvUSD (Stablecoins)",
+    symbol: "vcrvUSD_Stablecoins",
+    decimals: 8,
+    underlying: tokens.crvUSD,
+    exchangeRate: parseUnits("1", 28),
+    comptroller: COMPTROLLER_STABLECOINS,
+  },
+
+  // Curve Pool
+  vcrvUSD_Curve: {
+    name: "Venus crvUSD (Curve)",
+    symbol: "vcrvUSD_Curve",
+    decimals: 8,
+    underlying: tokens.crvUSD,
+    exchangeRate: parseUnits("1", 28),
+    comptroller: COMPTROLLER_CURVE,
+  },
+  vCRV_Curve: {
+    name: "Venus CRV (Curve)",
+    symbol: "vCRV_Curve",
+    decimals: 8,
+    underlying: tokens.CRV,
+    exchangeRate: parseUnits("1", 28),
+    comptroller: COMPTROLLER_CURVE,
+  },
+};
+
+interface RiskParameters {
+  borrowCap: string;
+  supplyCap: string;
+  collateralFactor: string;
+  liquidationThreshold: string;
+  reserveFactor: string;
+  initialSupply: string;
+  vTokenReceiver: string;
+}
+
+const riskParameters: { [key in VTokenSymbol]: RiskParameters } = {
```

Comment:
I would refactor to import these values from the VIP so it is obvious that they are same values